source/MRMesh/MRObjectLoad.cpp

```diff
@@ -289,6 +290,27 @@ Expected<std::vector<std::shared_ptr<MR::Object>>, std::string> loadObjectFromFi
         else
             result = unexpected( res.error() );
     }
+    else if ( ext == "*.zip" )
```

Comment:
why special if block i think zip should work in this if statement
`else if ( std::find_if( SceneFileFilters.begin(), SceneFileFilters.end(), [ext] ( const auto& filter ) { return filter.extensions.find( ext ) != std::string::npos; }) != SceneFileFilters.end() )`

Reply:
Refactored.